<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChefMainWindow</class>
 <widget class="QMainWindow" name="ChefMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1565</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addRecipeButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>240</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Recipe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myRecipesButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>350</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>My Recipes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addIngredientButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>450</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Ingredient</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1565</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
